Resources/Prototypes/_Goobstation/MartialArts/carp.yml

```
@@ -0,0 +1,64 @@
+- type: martialArt
+  id: SleepingCarp
+  martialArtsForm: SleepingCarp
+  roundstartCombos: SleepingCarpMoves
+  randomDamageModifier: True
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)